src/utils/graph/layout.js

```diff
@@ -337,7 +275,7 @@ const expandDenseRows = (edges, rows, spaceY) => {
     const density = densities[i];
 
     // Snap to improve vertical rhythm
-    const offsetY = snap(density * spaceY, Math.round(spaceY * 0.25));
+    const offsetY = snap(density * 1.25 * spaceY, Math.round(spaceY * 0.25));
```

Comment:
Yeah I know what you mean. As much as I'd like to often the problem is these kind of constants tend to be really abstract and highly implementation specific, so I tend to avoid making them appear like they are easily 'configurable' unless I know they are very well defined. In this case I gave it a shot though, so hopefully it helps.